financial-templates-lib/price-feed/CreatePriceFeed.js

```diff
@@ -197,6 +197,16 @@ const defaultConfigs = {
       { type: "cryptowatch", exchange: "poloniex", pair: "compusdt" },
       { type: "cryptowatch", exchange: "ftx", pair: "compusd" }
     ]
+  },
+  "COMP/USD": {
```

Comment:
I added this additional config as the Kovan deployment has a different identifier with the `/` in it :)